worlds/rabi_ribi/options.py

```diff
@@ -105,11 +124,37 @@ class UnderwaterWithoutWaterOrb(Toggle):
     """
     display_name = "Underwater Without Water Orb"
 
-class PlurkwoodReachable(Toggle):
+class IncludePlurkwood(Toggle):
     """
     If this flag is true, the game might expect you to go into Plurkwood.
     """
-    display_name = "Plurkwood Reachable"
+    display_name = "IncludePlurkwood"
+
+class IncludeWarpDestination(Toggle):
+    """
+    If this flag is true, locations in the warp destination area are included in the pool.
+    Note that some of these locations may require beating the game to reach again if missed.
```

Comment:
I wonder how feasible it is to add custom warps to these destinations in the future.

Reply:
That seems like it would work; I'll ask in the thread to see if there's a reason the standalone didn't do this.
To be clear, you don't need to make a new save or do NG+, you just have to beat the post game to go back to chapter 4(?) (I forget where the loop starts)